src/components/RevokeContainer/RevokeDocumentDropZone/RevokeDocumentDropZone.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { FunctionComponent, useState } from "react";
+import { RevokeDocumentDropZone } from "./RevokeDocumentDropZone";
+import { DocumentUploadState } from "../../../constants/DocumentUploadState";
+import { MemoryRouter } from "react-router";
+
+export default {
+  title: "Revoke/RevokeDocumentDropZone",
+  component: RevokeDocumentDropZone,
+  parameters: {
+    componentSubtitle: "RevokeDocumentDropZone",
+  },
+};
+
+export const Default: FunctionComponent = () => {
+  const [revokeDocument, setRevokeDocument] = useState();
+  const [fileName, setFileName] = useState("");
+  const [documentUploadState, setDocumentUploadState] = useState(DocumentUploadState.INITIALIZED);
+  console.info(revokeDocument, fileName);
```

Comment:
this is intended?

Reply:
ah sorry, this is stories, ignore

Reply:
yes for the stories, cos the unused state will throw an error